### ISSUE TITLE
Gratipay Team

### ISSUE THREAD

**Issue by dougwilson** [jshttp/jshttp.github.io#66]
So I'm looking into putting together a Gratipay team for jshttp, after the restructuring of Gratipay. I think that, in general, a team makes more sense anyway. It is the reason why no jshttp modules have a Gratipary badge, because before it would have to have been an individual's badge, rather than that of a team.

So who here wants to be on the Gratipay team, if approved, @jshttp/owners ?

For me to apply for it, I need to put the following information together, which I will do here, and if anyone wants to help, that would be awesome :)

Product or Service
> What product or service does your team provide?

Revenue Model
> How does your team make money?

Contributing
> How can other people get involved with your team?

Paying Contributors
> How do you share revenue with contributors?

**Comment by Fishrock123** [comment]
I don't really do that much around here right now, so I'll pass. 

:+1: for the idea though!

**Comment by dougwilson** [comment]
> What product or service does your team provide?

The jshttp team creates and maintains many different HTTP-related modules for the Node.js runtime. These modules are heavily uses as dependencies in various major projects like Express.js, Koa.js, Hapi.js and many others.

> How does your team make money?

The jshttp team is a collective and does not generate money in it's own right. Currently Gratipay is used to provide a path for interested users in contributing back to the team with money when contributing in other methods is not possible or viable.

> How can other people get involved with your team?

The jshttp team welcomes new contributors and even additional owners. We aim to be a community in which anyone can positively participate regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).­ Further details available at https://github.com/jshttp/style-guide/blob/master/template/Contributing.md

> How do you share revenue with contributors?

\<TBD\>

**Comment by jonathanong** [comment]
i'm pretty sure the team's just you now. haha. i'm just waiting on a few things that you've assigned yourself but other than that, it's just maintenance here and there.

**Comment by dougwilson** [comment]
The jshttp application is in and they are reviewing in https://github.com/gratipay/inside.gratipay.com/issues/211

**Comment by jonathanong** [comment]
i don't really care about this. @dougwilson you can have all the money :) i started to find asking for tips annoying

**Comment by Fishrock123** [comment]
yeah, agree with @jonathanong

**Comment by dougwilson** [comment]
Yea, they are still going back and forth on the whole thing, so I don't see getting together a team together any time soon. So far I've just been using it to pay for the `jshttp` and `pillarjs` parked domains, which eventually will be transferred to the Node.js foundation anyway.
